### DISCUSSION TITLE
Would it be possible to make a hotfix release for 1.3.1

### DISCUSSION THREAD

**Discussion by nre-ableton** [chocolatey/chocolatey-ansible#97] [General]
Issue #91 is quite disruptive, it basically causes the module to fail on any package that is pinned. Given that we pin _all_ of our packages, we can't update to 1.3.0.

Given that the [1.4.0 milestone](https://github.com/chocolatey/chocolatey-ansible/milestone/7) has a few other issues that are incomplete, would it be possible to make a hotfix and release 1.3.1? It would be much appreciated for those of us who can't use 1.3.0.

Thanks!

**Comment by pauby** [comment]
Can you elaborate more on this? Issue #91 is fixing a bug.

**Comment by nre-ableton** [comment]
@pauby Yes, I meant a fix to include https://github.com/chocolatey/chocolatey-ansible/pull/93 (ie, 217bfae154379d4f969eaf5bbfb2c7f3da5f91fa).

**Reply by pauby** [reply]
Ahhh. Leave it with me.

**Reply by nre-ableton** [reply]
Great, thanks!

**Comment by Windos** [comment]
@nre-ableton, working on a bugfix release atm. Should be out in the next day or so. I'll update here once it is available.

**Reply by Windos** [reply]
chocolatey.chocolatey v1.3.1 is now available on Ansible Galaxy!

**Reply by nre-ableton** [reply]
Thanks so much! @Windos feel free to close this discussion (I don't seem to have the ability to do that).
